json-lint/test/resources/schema/inventory/valid/adsense.chunk.test.json

```diff
@@ -0,0 +1,17 @@
+{
+  "placements": {
+    "inline": {
+      "adServer": "chunked"
+    }
+  },
+  "adServers": {
+    "chunked": {
+      "type": "adsense",
+      "ad-client": "ca-pub-7484406166969543",
+      "ad-slot": 9667904487,
+      "width": 300,
+      "height": 600,
+      "chunkFilename": "Adsense3"
```

Comment:
Why do we need this? We should work without this, for now this is produced by Gutenberg class AdServer.java. In any case this is not a valid AMP parameter and it should be put inside the json parameter. Can we discuss this offline?